clienttools/IDEPlugins/ESDL/esdl.bat.in

```diff
@@ -32,11 +32,8 @@ echo New Input file:  %newinput% >> %TMP%\log.txt
 echo New Ouput file:  %newoutput% >> %TMP%\log.txt
 copy /Y %3 %newinput%
 
-set "clienttoolsdir=@ORIG_PREFIX@\@DIR_NAME@\@version@\clienttools"
-set "clienttoolsbindir=%clienttoolsdir%\bin"
-set "clienttoolscompdir=%clienttoolsdir%\componentfiles"
-
-echo clienttoolsdir: %clienttoolsdir% >> %TMP%\log.txt
+set "clienttoolsbindir=.\..\bin"
```

Comment:
Does eclide guarantee that the current directory is set where you expect it to be? I assume that is the location of the .bat file ?

@GordonSmith ?

Reply:
This batch file is being installed in the client tools installer, so the relative path has to be correct.

Reply:
That's not really the question I was asking though... When the ide executes the batch file, what is the current directory? ../bin relative to the location of the batch file may be guaranteed to be ok, but that's not going to help us if the current directory is not set to the esdl.bat location before it is executed.

Reply:
Yes the IDE sets the working folder to the same folder the bat file is in.